config/Ruby/pry_repl.rb

```diff
@@ -1,6 +1,7 @@
 require 'rubygems'
 gem 'pry'
```

Comment:
Because there are bunch of fixes and new features in the latest version? See https://github.com/pry/pry/blob/master/CHANGELOG.md

Why force people to use old versions when newer stable versions are available for several months already?